opengrok-tools/src/main/python/opengrok_tools/utils/log.py

```diff
@@ -0,0 +1,85 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# See LICENSE.txt included in this distribution for the specific
+# language governing permissions and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at LICENSE.txt.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+
+#
+# Copyright (c) 2018, Oracle and/or its affiliates. All rights reserved.
+#
+
+import logging
+import sys
+
+
+class MinLogLevelFilter(logging.Filter):
```

Comment:
Please see https://docs.python.org/3/library/logging.html#filter-objects:

> Changed in version 3.2: You don’t need to create specialized Filter classes, or use other classes with a filter
> method: you can use a function (or other callable) as a filter. 

So maybe you don't need these classes at all?

Reply:
good point. works fine with lambdas.